omod/src/test/resources/org/openmrs/module/idgen/include/TestData.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+
+	<IDGEN_IDENTIFIER_SOURCE ID="1" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a901" NAME="Test Sequential Generator" IDENTIFIER_TYPE="1" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="2" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a902" NAME="Test Remote Source" IDENTIFIER_TYPE="2" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+	<IDGEN_IDENTIFIER_SOURCE ID="3" UUID="0d47284f-9e9b-4a81-a88b-8bb42bc0a903" NAME="Test Identifier Pool" IDENTIFIER_TYPE="1" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>
+    <IDGEN_IDENTIFIER_SOURCE ID="4" UUID="0d42284f-9e9b-4a81-a88b-8bb42bc0a904" NAME="Another Test Identifier Pool" IDENTIFIER_TYPE="4" CREATOR="1" DATE_CREATED="2009-11-30 11:28:30.489" RETIRED="false"/>

```

Comment:
Is there any reason why you have two dataset files instead of just one?

Reply:
deleted the second one.

Reply:
Which one did you delete?

Reply:
Did you address the above?

Reply:
yes

Reply:
How did you address it?

Reply:
Fixed. Removed the second test file.